### DISCUSSION TITLE
How to install a specific rust-analyzer version

### DISCUSSION THREAD

**Discussion by cyejing** [mason-org/mason.nvim#1024] [Q&A] [answered]
How to install a specific rust-analyzer version

I want to install an older version of rust-analyzer, not the latest version
The new version has compatibility issues with rust

**Comment by yutkat** [comment, marked as answer]
```
:MasonInstall rust-analyzer@nightly
```

https://github.com/williamboman/mason.nvim/blob/5b9fd3822d686092c7ee08adfcbd2c764def22c5/doc/mason.txt#L94-L97

**Comment by cyejing** [comment]
thanks, this is very helpful for me
